Automation/00_hydra/Sweden.R

```diff
@@ -123,84 +115,115 @@ if (date_f > last_date_drive){
     ) %>% 
     select(Country, Region, Code, Date, Sex, Age, AgeInt, Metric, Measure, Value)
   
+  data_source_zip <-  data_source
+  
   # 3.2. Vaccines ================
   
-  vac_sex <- read_xlsx(data_source_vac, sheet = 4)
-  vac_age <- read_xlsx(data_source_vac, sheet = 3)
+  # First, see if there is new vaccine data
   
-  # Get data by sex
+  data_source_vac <- paste0(dir_n, "Data_sources/", ctr, "/vaccination_",today(), ".xlsx")
   
-  vac_s2 <-
-    vac_sex %>% 
-    select(
-      Sex = starts_with("K")
-      , Value = `Antal vaccinerade`
-      , Measure = Dosnummer
-    ) %>% 
-    # filter(!grepl("^t", Sex, ignore.case = T)) %>% 
-    mutate(
-      Sex = case_when(
-        grepl("^m", Sex, ignore.case = T) ~ "m",
-        grepl("^k", Sex, ignore.case = T) ~ "f"
-        # grepl("^t", Sex, ignore.case = T) ~ "UNK"
-      ) 
-      , Measure = case_when(
-        grepl("1", Measure, ignore.case = T) ~ "Vaccination1"
-        , grepl("2", Measure, ignore.case = T) ~ "Vaccination2"
-      )
-      , Age = "TOT"
-      , AgeInt = ""
-      # Add empty row for UNK
-      , Sex = ifelse(is.na(Sex), "UNK", Sex)
-      , AgeInt = ifelse(Sex == "UNK", "", AgeInt)
-      , Value = ifelse(Sex == "UNK", 0, Value)
-    ) %>% 
-    select(Sex, Age, AgeInt, Measure, Value) %>% 
-    arrange(Sex)
+  url_vac <- "https://fohm.maps.arcgis.com/sharing/rest/content/items/fc749115877443d29c2a49ea9eca77e9/data"
+  httr::GET(url_vac, write_disk(data_source_vac))
   
-  # Get data by age
   
-  vac_a2 <-
-    vac_age %>% 
+  date_f_vac <- read_xlsx(data_source, sheet = 1) %>% 
```

Comment:
`data_source_vac` ?

Reply:
yes, werll spotted, thanks!

Reply:
did you already push a commit with this or shall I make those changes. Sorry this has been sitting here

Reply:
(FYI if you push a commit it should automatically join this PR)